fms/models/__init__.py

```diff
@@ -160,9 +160,7 @@ def _fsdp_wrap(
     # initializes parameters that are on meta devices
     def init_fn(x: nn.Module):
         if not rank0:
-            return x.to_empty(device=device, recurse=False)
-        else:
-            return x
+            x.to_empty(device=device, recurse=False)
```

Comment:
did we test that the return isn't required? I thought I had run into an issue without it

Reply:
https://pytorch.org/docs/stable/fsdp.html - if you look at where `param_init_fn` is defined, the docs show it must work inplace

Reply:
fair... let me check the actual code that uses this

Reply:
and in any case I'll revert it and leave it for another PR